### PR TITLE
Added S3 bucket definition for parsed course data from OCW

### PR DESCRIPTION
**Pull request by blarghmatey**

### CODE REVIEW

File: salt/orchestrate/aws/s3_buckets/operations.sls [hunk 1]
```diff
@@ -0,0 +1,11 @@
+open-learning-course-data:
+  boto_s3_bucket.present:
+    - Bucket: open-learning-course-data
+    - Versioning:
+        Status: Enabled
+    - region: us-east-1
+    - Tagging:
+        OU: operations
+        business_unit: operations
```
Comment (shaidar) [inline salt/orchestrate/aws/s3_buckets/operations.sls]: Should we have an `open` or `discussions` business_unit and use here?

Comment (blarghmatey) [reply salt/orchestrate/aws/s3_buckets/operations.sls]: I was wondering about that too and it's probably worth getting @pdpinch's input on the matter.

**Comment by shaidar** [comment]
It was decided to add a business unit for `open` to keep better track of those resources

**Comment by shaidar** [comment]
👍

**Pull request merged**